internal/bminventory/inventory.go

```diff
@@ -237,6 +261,96 @@ const nodeIgnitionFormat = `{
   }
 }`
 
+const configStaticIpsScript = `
```

Comment:
Consider moving it to a different file.

Reply:
i will leave it here for this commit, but will probably move it to the ignition for the next one (MGMT-3297)

Reply:
OK, then please add a todo comment so we will remember it.

Reply:
done